src/rotation.jl

```diff
@@ -0,0 +1,23 @@
+"""
+    VerticalRotationAxis{T} <: AbstractRotation
+
+A parameter object for constant rotation around a vertical axis.
+"""
+struct VerticalRotationAxis{T} <: AbstractRotation
+    f :: T
+end
+
+"""
+    VerticalRotationAxis([T=Float64;] f)
+
+Returns a parameter object for constant rotation at the angular frequency
+`2f`, and therefore with background vorticity `f`, around a vertical axis.
+
+Also called `FPlane`, after the "f-plane" approximation for the local effect of 
+Earth's rotation in a planar coordinate system tangent to the Earth's surface.
+"""
+function VerticalRotationAxis(T::DataType=Float64; f) 
+    return VerticalRotationAxis{T}(f)
+end
+
+const FPlane = VerticalRotationAxis
```

Comment:
Maybe it's a GFD thing but would `FPlane` be a better default name than `VerticalRotationAxis`? I find it's pretty common to say f-plane but not very common to say vertical rotation axis to describe a constant f scenario.

Reply:
That's fine with me. I started out that way, but decided that `VerticalRotationAxis` was more descriptive and precise and thus deserved to fill the primary role here (similar to using `Value` instead of `Dirichlet`). But if you would like to change back to `FPlane`, go ahead.